config/100-namespace.yaml

```diff
@@ -17,5 +17,4 @@ kind: Namespace
 metadata:
   name: knative-samples
   labels:
-    istio-injection: enabled
```

Comment:
Let's drop this change from here. It has nothing to do the the webhook change, or does it?